public/plugin.ts

```diff
@@ -0,0 +1,47 @@
+import { i18n } from '@kbn/i18n';
```

Comment:
unused import

Reply:
I will leave it for Yuxiang to fix all code in `public` dir :)